index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css"
+      integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="icon" href="./assets/coding.png" type="image/x-icon" />
+  </head>
+  <body>
+    <header>
+      <a id="logo" href="#">Welcome</a>
+      <div class="nav-desktop"></div>
+
+      <div class="nav-mobile">
+        <nav class="btn"><i class="fa-solid fa-bars"></i></nav>
+      </div>
+    </header>
+
+    <section class="welcome-section">
+      <div class="welcome-header-text">
+        <h1>Hey there. I'm Kano</h1>
+        <h2 class="second-header">I am software developer</h2>
+      </div>
+      <div class="intro-text">
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-links">
+          <li>
+            <a href="https://twitter.com/Kanombola_s">
+              <img
+                src="./assets/social-media/twitter.svg"
+                alt="twitter social media icon "
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://www.linkedin.com/in/kanombola-kanombola-a38b061a4/"
+            >
+              <img
+                src="./assets/social-media/Linkedin.svg"
+                alt="github social media icon "
+            /></a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                src="./assets/social-media/medium.svg"
+                alt="Medium social media icon "
+            /></a>
+          </li>
+          <li>
+            <a href="https://github.com/KanoCode"
+              ><img
+                src="./assets/social-media/github.svg"
+                alt="github social media icon "
+            /></a>
+          </li>
+
+          <li>
+            <a href="#"
+              ><img
+                src="./assets/social-media/angellist.svg"
+                alt="angellist social media icon "
+            /></a>
+          </li>
+        </ul>
+      </div>
+
+      <div class="next-page">
+        <img
+          src="./assets/page-navigator.svg"
+          alt="button to go to the next page"
+        />
+      </div>
+    </section>
+    <section class="work-section">
+      <div class="section-title">
+        <!-- <h2></h2> -->
+        <span></span>
```

Comment:
- [ ] Please remove inline comments from your code. It looks like your code is messy. :+1: